src/PhpSpreadsheet/Style/Conditional.php

```diff
@@ -28,6 +28,9 @@ class Conditional implements IComparable
     const OPERATOR_NOTCONTAINS = 'notContains';
     const OPERATOR_BETWEEN = 'between';
 
+    const IF_TRUE_STOP = true;
+    const IF_TRUE_DONT_STOP = false;
```

Comment:
I meant those two `const` should be deleted entirely. There is no need to define a const for `true`.